docs/developer-notes.md

```diff
@@ -0,0 +1,29 @@
+# Fehlende data-testid
+## Erläuterung
+Der Playwright-Codegenerator ist ein sehr wirkungsvolles tool um schnell Ausdrücke für die Elemente auf einer Webseite automatisch zu erstellen.
+Der FE-Entwickler des Clients muss das Attribut "data-testid" für jedes Element einfügen, welches für die Playwright-Testautomatisierung relevant ist.
+Welche Elemente relevant sind, weiss der Entwickler zum Zeitpunkt der Entwicklung des FE nicht und kann deshalb die data-testids nur erraten.
+
+## Hinweise für die Vergabe einer data-testid im FE
+Die data-testid muss genau in dem Bereich liegen, in dem der Endanwender auch tatsächlich den Klick ausführt, oder auch dort wo das Auge des Anwenders hinschaut(auf die Texte)
+
+## Liste der fehlenden oder nicht korrekt plazierten data-testid
+## menu.page.ts
+"NAVIGATION": div class="v-list-item-title">Navigation</div>
```

Comment:
Es gibt bereits eine Id `data-testid="menu-bar-title"` am parent div. Da Vuetify teilweise die DOM-Elemente selbst generiert, haben wir in solchen Fällen keine Chance, die Ids zu 100% an die Elemente mit den Inhalten zu setzen. Das wird alle Menüpunkte der Menu Bar betreffen. Ich schaue explizit bei den genannten Fällen, ob etwas an den Parent divs fehlt, aber ansonsten bitte die Syntax `[data-testid="...."] .item-list-title` verwenden.

Reply:
Habe die Test Ids für die Überschriften ergänzt, bitte einmal prüfen: https://github.com/dBildungsplattform/schulportal-client/pull/80

Reply:
Die Test Ids sind jetzt alle in Playwright angepast